var/spack/repos/builtin/packages/cquery/package.py

```diff
@@ -18,6 +18,9 @@ class Cquery(CMakePackage):
 
     depends_on('llvm')
 
+    # trivial patch (missing header) by mueller@kip.uni-heidelberg.de
```

Comment:
Normally I'd ask for a link to a discussion, but the site for `cquery` mentions that it's no longer under development, so this seems fine.